src/dioptra/restapi/v1/groups/controller.py

```diff
@@ -52,9 +67,28 @@ def get(self):
         log = LOGGER.new(
             request_id=str(uuid.uuid4()), resource="Group", request_type="GET"
         )
-        log.debug("Request received")
         parsed_query_params = request.parsed_query_params  # noqa: F841
 
+        search_string = parsed_query_params["search"]
+        page_index = parsed_query_params["index"]
+        page_length = parsed_query_params["page_length"]
+
+        groups, total_num_groups = self._group_service.get(
+            search_string=search_string,
+            page_index=page_index,
+            page_length=page_length,
+            log=log,
+        )
+        return utils.build_paging_envelope(
+            "groups",
+            utils.build_group,
+            groups,
+            search_string,
+            page_index,
+            page_length,
+            total_num_groups,
+        )
+
```

Comment:
Follow convention to provide explicit keyword arguments for 2nd position onward:

```suggestion
        return utils.build_paging_envelope(
            "groups",
            build_fn=utils.build_group,
            data=groups,
            query=search_string,
            index=page_index,
            length=page_length,
            total_num_elements=total_num_groups,
        )

```